src/features/force-update-modal/force-update-modal.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Modal, Text, View } from 'react-native';
+
+import { isUpdated, openStore } from 'hooks/force-update';
+
+import { translate } from 'localization/hooks';
+
+import { styles } from './force-update-modal.styles';
+
+interface ForceUpdateProps {}
+
+const ForceUpdateModal = ({}: ForceUpdateProps) => {
+  const [showModal, setShowModal] = useState(false);
+
+  useEffect(() => {
+    const checkAndUpdate = async () => {
+      const updated = await isUpdated();
+      if (!updated) {
+        setShowModal(true);
+      }
+    };
+    checkAndUpdate();
+  }, []);
```

Comment:
You can save this `useEffect` if you move it within the custom hook, I'll get back to you offline about this.

Reply:
Done